internal/bundle/fmt.go

```diff
@@ -68,13 +68,18 @@ func sanitizeStringForYaml(s string) string {
 	return strings.Join(lines, "\n")
 }
 
-func FormatFile(filename string) error {
+func FormatAndSortFile(filename string, sortFields []string) error {
```

Comment:
I would keep the name simple, the method signature changes anyway. `Format(filename string, sort bool)` sounds like it could do the trick